src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2688,8 +2787,8 @@ enum VertexType {
     """TRANSIT"""
     TRANSIT
 
-    """BIKEPARK"""
```

Comment:
This breaks backwards compatability a bit. We could leave this as it was with the BIKEPARK and PARKANDRIDE, and add a new type with those replaced with VEHICLEPARKING (and at the same time rename BIKESHARE as VEHICLERENTAL). Although, I'm not quite sure what are all the use cases for this vertexType. Maybe it makes sense to be able to differentiate between bike and car parks based on this value but in theory, now they can have both cars and bicycles.

Reply:
`PARKANDRIDE` is not present in `VertexType`s, so it wouldn't have been returned in the API. I've renamed `VEHICLERENTAL` to `BIKEPARK` which should preserve the backward compatibility.